terminus-ui/autocomplete/src/autocomplete.component.ts

```diff
@@ -142,33 +143,44 @@ export class TsAutocompleteComponent implements OnInit,
 
   /**
    * Give the component an explicit name
+   *
+   * @internal
    */
   public readonly componentName = 'TsAutocompleteComponent';
 
   /**
    * Define the FormControl
+   *
+   * @internal
    */
   public autocompleteFormControl = new FormControl([]);
 
   /**
    * Store a reference to the document object
+   *
+   * @internal
    */
   private document: Document;
 
   /**
    * Subject used to alert the parent {@link TsFormFieldComponent} when the label gap should be recalculated
    *
    * Implemented as part of TsFormFieldControl.
+   * @internal
```

Comment:
I see sometimes you have a line break before `@internal` sometimes you don't.... does that matter?

Reply:
It's not about the flag itself, but there should always be a line break after the JSDoc definition or the text will be generated on a single line

```typescript
/**
 * Line one
 * this will be generated on line one
 * me too
 */

/**
 * Line one
 *
 * Now I will be line #2 when generated
 */
```

Reply:
I guess I meant does a line break before the flag needed?
```
/**
 * Line
 * @internal
 */
```
vs.
```
/**
 * Line
 *
 * @interal
 */
```

Reply:
Nope. In your examples it makes no difference if the second line is a flag or plain text. Line #2 must _always_ have a line break.

```typescript
// good
/**
 * Line one
 *
 * Must have a line break for the next piece of docs.
 */

// good
/**
 * Line one
 *
 * @tagOne
 * @tagTwo
 */

// good
/**
 * Line one
 *
 * Line two
 * @tagOne
 */

// bad
/**
 * Line one
 * @tagOne
 * @tagTwo
 */

// bad
/**
 * Line one
 * Line two
 */
```